lib/codeowners/cli/main.rb

```diff
@@ -61,25 +60,64 @@ def write_codeowners
       end
 
       def suggest_add_to_codeowners(file)
-        return unless yes?("File added: #{file.inspect}. Add owner to the CODEOWNERS file?")
+        return if @quit
 
-        owner = ask('File owner(s): ')
-        new_line = create_new_pattern(file, owner)
-
-        unless new_line.pattern?
-          puts "#{owner.inspect} is not a valid owner name."
-          return
+        case add_to_codeowners_dialog(file)
+        when 'y' then add_to_codeowners(file)
+        when 'i' then return
+        when 'q' then @quit = true
         end
+      end
+
+      def add_to_codeowners_dialog(file)
+        ask(<<~QUESTION, limited_to: %w[y i q])
+          File added: #{file.inspect}. Add owner to the CODEOWNERS file?
+          (y) yes
+          (i) ignore
+          (q) quit
```

Comment:
this will lead to saving the file? should we add a note `quit (and save)`

do we want to abort to not save the file? maybe we should add `(CTRL-C) abort` ;)

Reply:
We save it